use-timescale/integrations/query-admin/psql.md

```diff
@@ -1,224 +1,267 @@
 ---
-title: Install the psql connection tool
-excerpt: How to install the psql client for PostgreSQL
+title: Connect to a Timescale Cloud service with psql 
+excerpt: Install the psql client for PostgreSQL and connect to your service 
 products: [cloud, mst, self_hosted]
 keywords: [connect, psql]
 ---
 
-# Install the psql connection tool
+import IntegrationPrereqs from "versionContent/_partials/_integration-prereqs.mdx";
 
-The `psql` command line tool is widely used for interacting with a PostgreSQL or
-Timescale instance, and it is available for all operating systems. Most of
-the instructions in the Timescale documentation assume you are using `psql`.
+# Connect with psql
 
-Before you start, check that you don't already have `psql` installed. It is
-sometimes installed by default, depending on your operating system and other
-packages you have installed over time:
+You use the `psql` command line tool to interact with your $SERVICE_LONG.
 
-<Terminal>
+## Prerequisites
+
+<IntegrationPrereqs />
+
```

Comment:
Can you add a line here saying you can also use psql to connect to a self-hosted deployment of TimescaleDB.